bucket/keyferret.json

```
@@ -0,0 +1,16 @@
+{
+    "version": "3.0",
+    "description": "Type accents, symbols, maths, Cyrillic, Greek, Hebrew and IPA in any Windows application",
+    "homepage": "https://www.keyferret.com/",
+    "license": "MIT",
+    "url": "https://www.keyferret.com/download/7/#/dl.zip",
+    "hash": "9ebef07ef41a31a8afef16e664d6a0b1f7dfdc39338f24379a44d5fd4c757489",
+    "extract_dir": "keyferret",
+    "bin": "keyferret.exe",
+    "shortcuts": [
+        [
+            "keyferret.exe",
+            "Keyferret"
+        ]
+    ]
```

```suggestion
    ],
    "persist": [
        "user.kflf",
        "global.kflf"
    ],
    "checkver": {
        "url": "https://www.keyferret.com/get/",
        "regex": "title=\"Version\\s([\\d.]+)"
    },
    "autoupdate": {
        "url": "https://www.keyferret.com/download/7/#/dl.zip"
    }
```

Also consider adding `persist`, `checkver` and `autoupdate`.
`persist` keeps the config file untouched when user updates the software through *Scoop*.